sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v2/query.sql

```diff
@@ -196,14 +196,14 @@ unioned AS (
   SELECT
     *,
     'shutdown' AS source_ping,
-    3 AS source_ping_priority
+    2 AS source_ping_priority
```

Comment:
Since this priority changes and differs from what's in the requirements & caveats docs, could we update the [inline comment](https://github.com/mozilla/bigquery-etl/blob/dffbb3afef5a3b0b5adc396fa4b874bc0f617b6b/sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v2/query.sql#L187) to explain the reasoning for future reference?

Reply:
Will update 👍

Reply:
Update, following up on this offline this change might not be needed

Reply:
I'll revert and fix this in a separate PR according to the discussion [here](https://docs.google.com/document/d/1LmfuThLp856-W0HDxyH9Lco4piVKW_6ok4mfnBnXOl4/edit?disco=AAAA_MAPsZw).